sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/statsbeat/_statsbeat.py

```diff
@@ -0,0 +1,108 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+import os
+import threading
+
+from opentelemetry.sdk.metrics import MeterProvider
+from opentelemetry.sdk.metrics.export import PeriodicExportingMetricReader
+
+from azure.monitor.opentelemetry.exporter.statsbeat._exporter import _StatsBeatExporter
+from azure.monitor.opentelemetry.exporter.statsbeat._statsbeat_metrics import _StatsbeatMetrics
+
+# pylint: disable=line-too-long
+_DEFAULT_NON_EU_STATS_CONNECTION_STRING = "InstrumentationKey=c4a29126-a7cb-47e5-b348-11414998b11e;IngestionEndpoint=https://westus-0.in.applicationinsights.azure.com/"
```

Comment:
We have so many constants here and in _state. What do you think of just moving all or most of them to a separate constants.py file. That should also make circular dependencies less of an issue.

Reply:
Not a blocker. Just a thought.

Reply:
Circular dependencies would still be an issue regardless (see above explanation). I believe it would be more Pythonic and easier to read to have all related constants in the same file (similar to [OpenTelemetry](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py#L71)). Also based off personal preference, I prefer not putting constants in a separate file that are not accessed by anywhere else in the codebase (things like `_DEFAULT_NON_EU_STATS_CONNECTION_STRING` will only ever be accessed by the file that it is in).

Reply:
Sounds good to me.